modules/gameSearch/src/main/Query.scala

```diff
@@ -54,27 +54,27 @@ object Query {
 
   import lila.common.Form._
   import play.api.libs.json._
+  import play.api.i18n.Lang
+  import lila.i18n.{ I18nKeys => trans }
 
   import Range.rangeJsonWriter
   implicit private val sortingJsonWriter  = Json.writes[Sorting]
   implicit private val clockingJsonWriter = Json.writes[Clocking]
   implicit val jsonWriter                 = Json.writes[Query]
 
-  val durations: List[(Int, String)] =
-    ((30, "30 seconds") ::
+  def durations(implicit lang: Lang): List[(Int, String)] =
+    ((30, trans.nbSeconds.pluralSame(30).render) ::
```

Comment:
`trans.nbSeconds.pluralSameTxt(30)` to avoid translation XSS!

Reply:
done